From davej@redhat.com  Wed Mar  1 20:21:09 2006
Date: Wed, 1 Mar 2006 23:10:31 -0500
From: Dave Jones <davej@redhat.com>
To: Chris Wright <chrisw@sous-sol.org>
Cc: stable@kernel.org
Subject: [PATCH] mempolicy.c compile fix, make sure BITS_PER_BYTE is defined

Gar..

mm/mempolicy.c: In function 'get_nodes':
mm/mempolicy.c:527: error: 'BITS_PER_BYTE' undeclared (first use in this function)
mm/mempolicy.c:527: error: (Each undeclared identifier is reported only once
mm/mempolicy.c:527: error: for each function it appears in.)

About to retry a build with the below patch which should do the trick.
(How did this *ever* build?)

Signed-off-by: Dave Jones <davej@redhat.com>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
---

 include/linux/types.h |    1 +
 1 files changed, 1 insertion(+)

--- linux-2.6.15.5.orig/include/linux/types.h
+++ linux-2.6.15.5/include/linux/types.h
@@ -8,6 +8,7 @@
 	(((bits)+BITS_PER_LONG-1)/BITS_PER_LONG)
 #define DECLARE_BITMAP(name,bits) \
 	unsigned long name[BITS_TO_LONGS(bits)]
+#define BITS_PER_BYTE 8
 #endif
 
 #include <linux/posix_types.h>
